chall03/tkuumola.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import(
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strconv"
+	"time"
+)
+
+func main(){
+	start := time.Now()
+	resp, err := http.Get("https://chall03.hive.fi/")
+	if err != nil {
+		log.Fatalln(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	defer resp.Body.Close()
+	if err != nil{
+		log.Fatalln(err)
+	}
+	fmt.Println("GET	https://chall03.hive.fi/")
+	fmt.Printf("	answer: %s\n\n", string(body))
+
+	r := regexp.MustCompile(`\d+`)
```

Comment:
regex wasn't necessary and should be avoided for this kind of exercise: https://softwareengineering.stackexchange.com/questions/113237/when-you-should-not-use-regular-expressions